zeeguu/core/content_recommender/elastic_recommender.py

```diff
@@ -303,3 +302,78 @@ def _difficuty_level_bounds(level):
         lower_bounds = 4
         upper_bounds = 8
     return lower_bounds, upper_bounds
+
+
+def __find_articles_like(recommended_articles_ids: 'list[int]', limit: int, article_age: int, language_id: int) -> 'list[Article]':
+    es = Elasticsearch(ES_CONN_STRING)
+    fields = ["content", "title"]
+    language = Language.find_by_id(language_id)
+    like_documents = [
+        {"_index": ES_ZINDEX, "_id": str(doc_id) } for doc_id in recommended_articles_ids
+    ]
+
+    cutoff_date = datetime.now() - timedelta(days=article_age)
+
+    mlt_query = {
```

Comment:
I would suggest building this query by using a pattern similar of what is used in `build_elastic_recommender_query` under `elastic_query_builder.py`.

Reply:
@tfnribeiro 
Implemented your suggestion 👍